VRDR/DeathRecord.cs

```diff
@@ -6197,8 +6378,19 @@ private void RestoreReferences()
                 throw new System.ArgumentException("Failed to find a Certifier (Practitioner). The third entry in the FHIR Bundle is usually the Certifier (Practitioner). Either the Certifier is missing from the Bundle, or the attestor reference specified in the Composition is incorrect.");
             }
 
+            // Grab Pronouncer
+            /* Axiell Notes:
+             *  - not sure how to correctly get the Pronouncer entry here, since he/she is not an Attester (or should he/she be?)
+             *  - I don't think the pronouncer is always required, either, so there is no exception thrown when missing.
+            var pronouncerEntry = Bundle.Entry.FirstOrDefault(entry => entry.Resource.ResourceType == ResourceType.Practitioner && (entry.FullUrl == Composition..pron.Attester.First().Party.Reference || (entry.Resource.Id != null && entry.Resource.Id == Composition.Attester.First().Party.Reference)));
+            if (pronouncerEntry != null)
+            {
+                Pronouncer = (Practitioner)practitionerEntry.Resource;
+            }
+            */
+
             // Grab Mortician
-            var morticianEntry = Bundle.Entry.FirstOrDefault( entry => entry.Resource.ResourceType == ResourceType.Practitioner && ((Practitioner)entry.Resource).Id != Certifier.Id );
+            var morticianEntry = Bundle.Entry.FirstOrDefault( entry => entry.Resource.ResourceType == ResourceType.Practitioner && ((Practitioner)entry.Resource).Id != Certifier.Id && (Pronouncer != null && ((Practitioner)entry.Resource).Id != Pronouncer.Id));
```

Comment:
I think this might be the source of the unit test failures. The addition of the pronouncer clause is causing the code to fail to find the morticianEntry. To identify the pronouncer entry you may have to first get the id of the practitioner from the VRDR Death Date performer and then use that as a criteria for selecting the matching practitioner entry.